packages/lb-annotation/src/core/pointCloud/index.ts

```diff
@@ -299,6 +300,43 @@ export class PointCloud {
     }, {});
   }
 
+  /**
+   * add new sphere just like adding new box
+   * @param sphereParams
+   */
+  public AddSphereToSense = (sphereParams: IPointCloudSphere, color = 'blue') => {
+    const id = sphereParams.id ?? uuid();
+
+    this.removeObjectByName(id);
+
+    const { center, widthSegments, heightSegments, radius } = sphereParams;
+    const group = new THREE.Group();
+    const spGeo = new THREE.SphereGeometry(radius, widthSegments, heightSegments);
+    const spMaterial = new THREE.MeshBasicMaterial({ color });
+    const sphere = new THREE.Mesh(spGeo, spMaterial);
+    sphere.position.set(center.x, center.y, center.z);
+    group.add(sphere);
+    group.name = id;
+    this.scene.add(group);
+  };
+
+  public generateSphere = (sphereParams: IPointCloudSphere) => {
+    this.render();
+    const { fill } = toolStyleConverter.getColorFromConfig(
+      { attribute: sphereParams.attribute },
+      { ...this.config, attributeConfigurable: true },
+      {},
+    );
+    this.AddSphereToSense(sphereParams, fill);
+  };
+
+  public generateSpheres = (spheres: IPointCloudSphere[]) => {
+    spheres.forEach((sphere) => {
+      this.generateSphere(sphere);
```

Comment:
You can add all sphere and render it. (generateSphere will execute with render)

Reply:
done